src/main/kotlin/notary/NotaryInitialization.kt

```diff
@@ -71,7 +74,8 @@ class NotaryInitialization {
      */
     fun initIrohaConsumer() {
         logger.info { "Init Iroha consumer" }
-        irohaConsumer = IrohaConsumerImpl(notary)
+        val cryptoProvider = IrohaCryptoProviderImpl(Configs.pubkeyPath, Configs.privkeyPath)
```

Comment:
IrohaCryptoProvide has to get own dependencies e.g. keys directly. Code that loading keys from files should be used in initialization class explicitly.